minecraft_servers/craftedhorizon/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
     "server_name": "craftedhorizon",
     "nice_name": "CraftedHorizon",
-    "direct_ip": "CraftedHorizon.net",
-    "supported_languages": [
-        "de"
+    "direct_ip": "mc.craftedhorizon.net",
+    "server_wildcards": [
+      "%.craftedhorizon.net"
```

Comment:
Not a real wildcard
https://i.imgur.com/U8hD4P3.png

Reply:
ah you mean that